src/perf_counter/perf_counter_atomic.h

```diff
@@ -214,12 +219,17 @@ class perf_counter_number_percentile_atomic : public perf_counter
             "pegasus_perf_counter_number_percentile_atomic counters");
         _timer.reset(new boost::asio::deadline_timer(tools::shared_io_service::instance().ios));
         _timer->expires_from_now(
-            boost::posix_time::seconds(rand() % _counter_computation_interval_seconds + 1));
+            boost::posix_time::seconds(::rand() % _counter_computation_interval_seconds + 1));
```

Comment:
Why did you add the prefix :: ?

Reply:
> Why did you add the prefix :: ?

Without prefixal `::`, rDSN will be failed to be compiled, showing error message: **error: expected primary-expression before ‘(’ token**. Since it's been included by other source files such as `src/utils/test/nth_element_utils.h` and `src/utils/test/nth_element_bench/nth_element_bench.cpp`, `rand()` may has conflicted with some functions of other header files. Therefore, I've added a prefixal `::` to resolve.